sdk/devcenter/arm-devcenter/src/operations/catalogDevBoxDefinitions.ts

```diff
@@ -1,323 +0,0 @@
-/*
- * Copyright (c) Microsoft Corporation.
- * Licensed under the MIT License.
- *
- * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is regenerated.
- */
-
-import { PagedAsyncIterableIterator, PageSettings } from "@azure/core-paging";
-import { setContinuationToken } from "../pagingHelper";
-import { CatalogDevBoxDefinitions } from "../operationsInterfaces";
-import * as coreClient from "@azure/core-client";
-import * as Mappers from "../models/mappers";
-import * as Parameters from "../models/parameters";
-import { DevCenterClient } from "../devCenterClient";
-import {
-  DevBoxDefinition,
-  CatalogDevBoxDefinitionsListByCatalogNextOptionalParams,
-  CatalogDevBoxDefinitionsListByCatalogOptionalParams,
-  CatalogDevBoxDefinitionsListByCatalogResponse,
-  CatalogDevBoxDefinitionsGetOptionalParams,
-  CatalogDevBoxDefinitionsGetResponse,
-  CatalogDevBoxDefinitionsGetErrorDetailsOptionalParams,
-  CatalogDevBoxDefinitionsGetErrorDetailsResponse,
-  CatalogDevBoxDefinitionsListByCatalogNextResponse
-} from "../models";
-
-/// <reference lib="esnext.asynciterable" />
-/** Class containing CatalogDevBoxDefinitions operations. */
-export class CatalogDevBoxDefinitionsImpl implements CatalogDevBoxDefinitions {
-  private readonly client: DevCenterClient;
-
-  /**
-   * Initialize a new instance of the class CatalogDevBoxDefinitions class.
-   * @param client Reference to the service client
-   */
-  constructor(client: DevCenterClient) {
-    this.client = client;
-  }
-
-  /**
-   * List Dev Box definitions in the catalog.
-   * @param resourceGroupName The name of the resource group. The name is case insensitive.
-   * @param devCenterName The name of the devcenter.
-   * @param catalogName The name of the Catalog.
-   * @param options The options parameters.
-   */
-  public listByCatalog(
```

Comment:
why we have this removed?

Reply:
this api is added only in last preview release, not in last stable release